File: decimalToBinary.js
function decimalToBinary(n){
    let rem;
    let quot = n;
    let bin = ""

    while(quot !== 0){
        rem= quot % 2
        bin = rem + bin
        quot = Math.floor(quot / 2)
        
    }
    return bin
}

console.log(decimalToBinary(10))


let decimalNum = 25;
let binaryNum = parseInt(decimalNum, 10).toString(2);
console.log(binaryNum); // Output: 11001

File: decimalToBInary.java
public class decimalToBInary {
    public static void main(String[] args) {
        System.out.println(decimalToBinary(10));
    }

    public static String decimalToBinary(int n){
        String bin = "";
        int quot = n;
        int rem;

        while(quot != 0){
            rem = quot % 2;
            bin = rem + bin;
            quot = quot / 2;
        }

        return bin;
    }
}

